Fix incorrect statement regarding actor timers (#3977)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/actors/actors-features-concepts.md

Dapr actors are virtual, meaning that their lifetime is not tied to their in-memory representation. As a result, they do not need to be explicitly created or destroyed. The Dapr actor runtime automatically activates an actor the first time it receives a request for that actor ID. If an actor is not used for a period of time, the Dapr actor runtime garbage-collects the in-memory object. It will also maintain knowledge of the actor's existence should it need to be reactivated later.

> Invocation of actor methods and reminders reset the idle time, e.g. reminder firing will keep the actor active. Actor reminders fire whether an actor is active or inactive, if fired for inactive actor, it will activate the actor first. Actor timer firing does reset the idle time, however timers only fire while the actor is active.

Reviewer: ```suggestion
Invocation of actor methods, timers and reminders reset the actor idle time. For example a reminder firing keeps the actor active. 
- Actor reminders fire whether an actor is active or inactive and if fired for inactive actor, it activates the actor first.
- Actor timers firing reset the idle time, however timers only fire while the actor is active.
```
